app/va/va_profile/va_profile_client.py

```diff
@@ -50,6 +69,86 @@ def init_app(
         self.va_profile_token = va_profile_token
         self.statsd_client = statsd_client
 
+    def get_profile(self, va_profile_id: RecipientIdentifier) -> Profile:
+        """
+        Retrieve the profile information for a given VA profile ID using the v3 API endpoint.
+
+        Args:
+            va_profile_id (RecipientIdentifier): The VA profile ID to retrieve the profile for.
+
+        Returns:
+            Profile: The profile information retrieved from the VA Profile service.
+        """
+        recipient_id = transform_to_fhir_format(va_profile_id)
+        oid = OIDS.get(IdentifierType.VA_PROFILE_ID)
+        url = f'{self.va_profile_url}/profile-service/profile/v3/{oid}/{recipient_id}'
+        data = {'bios': [{'bioPath': 'contactInformation'}, {'bioPath': 'communicationPermissions'}]}
+
+        try:
+            response = requests.post(url, json=data, cert=(self.ssl_cert_path, self.ssl_key_path), timeout=(3.05, 1))
+            response.raise_for_status()
+        except (requests.HTTPError, requests.RequestException, requests.Timeout) as e:
+            self._handle_exceptions(va_profile_id.id_value, e)
+
+        response_json: Dict = response.json()
+        return response_json.get('profile', {})
+
+    def get_telephone_from_profile_v3(self, va_profile_id: RecipientIdentifier) -> str:
+        """
+        Retrieve the telephone number from the profile information for a given VA profile ID.
+
+        Args:
+            va_profile_id (RecipientIdentifier): The VA profile ID to retrieve the telephone number for.
+
+        Returns:
+            str: The telephone number retrieved from the VA Profile service.
+        """
+        contact_info: ContactInformation = self.get_profile(va_profile_id).get('contactInformation', {})
+        self.logger.info(f'V3 Profile - Retrieved ContactInformation: {contact_info}')
```

Comment:
We do not want to log contact info in prod. That can be in a debug statement (with a qualifier) or the whole statement can be made into a debug statement. 

No [f-strings in logger statements](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Process/style_guide.md#logging) please.